crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__all.snap

```diff
@@ -0,0 +1,174 @@
+---
+source: crates/sui/src/unit_tests/upgrade_compatibility_tests.rs
+expression: normalize_path(err.to_string())
+---
+error[Compatibility E01001]: missing public declaration
+  ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:6:18
+  │
+6 │ module upgrades::upgrades {
+  │                  ^^^^^^^^ enum 'EnumToBeRemoved' is missing
+  │
+  = enums are part of a module's public interface and cannot be removed or changed during an upgrade.
+  = add missing enum 'EnumToBeRemoved' back to the module 'upgrades'.
+
+error[Compatibility E01001]: missing public declaration
+  ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:6:18
+  │
+6 │ module upgrades::upgrades {
+  │                  ^^^^^^^^ struct 'StructToBeRemoved' is missing
+  │
+  = structs are part of a module's public interface and cannot be removed or changed during an upgrade.
+  = add missing struct 'StructToBeRemoved' back to the module 'upgrades'.
+
+error[Compatibility E01003]: ability mismatch
+   ┌─ /fixtures/upgrade_errors/all_v2/sources/UpgradeErrors.move:11:19
+   │
+11 │     public struct StructAbilityMismatchAdd has copy {} // added the copy ability where none existed
+   │                   ^^^^^^^^^^^^^^^^^^^^^^^^ Unexpected ability: 'copy'
+   │
+   = Structs are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original abilities of the struct: none for struct 'StructAbilityMismatchAdd'.
```

Comment:
Alternative wording -- the "none" seems easy to get lost in the other words here.

```suggestion
   = Restore the original abilities of struct 'StructAbilityMismatchAdd: none.
```